validation/dam_break_2d/exp_pressure_sensor_P1.csv

```diff
@@ -0,0 +1,99 @@
+time,P1
```

Comment:
IMHO, it might be cleaner to have all the pressure sensors in one file, no?

Reply:
They are manually extracted the times are not identical?

Reply:
ah nvm. There are already multiple columns for the different heights. It would be to complicated, right?

Reply:
So this would be time_P1, P1, time_P2, P2, time_P3, P3